formats/log/README.md

```diff
@@ -45,25 +45,42 @@ signatures, the format itself is not restricted to this scheme.
 The checkpoint body is of the form:
 
 ```text
-<Ecosystem & version string>
+<Origin string>
 <Decimal log size>
 <Base64 log root hash>
 [otherdata]
 ```
 
 The first 3 lines of the body **MUST** be present in all Checkpoints.
-`otherdata` is opaque and optional, and, if necessary, can be used to tie extra
-data to the checkpoint, however its format must conform to the sumdb signed
-note spec (e.g. it must not contain blank lines.)
+
+* `<Origin string>` should be a unique identifier for the log identity which issued the checkpoint.
+  The exact format of this is left undefined, but examples of types of value to put here
+  are the log's URL, a base64 encoded hash of its public key, etc.
+
+  The presence of this identifier forms part of the log claim, and prevents witness
+  signatures being moved between log checkpoints with identical contents, preventing a
+  particular type of targeted attack.
```

Comment:
This is true, but expanding this won't do us any harm. Something a little closer to a compelling explanation for me would be something like:

Including a unique log string in the checkpoint prevents distinct logs from creating checkpoints with the same contents. This prevents signatures from being transferred between checkpoints. As an example of an attack that this mitigates: a witness signature on a checkpoint confirms that the log's append-only property has been verified as of this checkpoint. If another log creates the same checkpoint, the witness signature can be used on that checkpoint which implies a verification check that may never have happened.

Reply:
I'm not at all against sharing the details, but I'm not sure whether this is the right place to put them - if I'm implementing this, I suspect that at a first pass I just want to know what the fields are and how I should use them, later on I might be interested to know why.

Would having these sorts of details in an issue or appendix section be better? (Maybe I should also remove the "preventing a particular type..." bit too?)

Reply:
Yeah I think dropping that removes the slippery slope into explaining the whole thing here. To keep this short and still accurate, perhaps "The presence of this identifier forms part of the log claim, and means that no two logs can create byte-equivalent checkpoints.".

Witnesses, signatures, and all of that can then be moved elsewhere.

Reply:
Reworded and remove the extra stuff for now - we can figure out where to put that later.